docs/pages/intro.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: "Cauldron React: Accessible Components Library"
+name: Introduction
+path: /
+---
+
+## Purpose
+
+Friends don’t let friends ship inaccessible code! These accessible packages include everything from typography and colors, to components like custom form controls. The design and interactions shown throughout this site are intended to show how Deque provides accessible experiences for the users of our own products - through the use of common, accessible components like these.
+
+## Usage
+
+Deque's design system contains two main libraries: Cauldron React, and Cauldron styles. Cauldron React is an accessible React component library, while Cauldron Styles contains the accessible styles for these components and more!
+
+### Installation
+
+To start using Cauldron React, install the packages with your package manager of choice:
+
+#### npm
+
+```js
+npm install @deque/cauldron-react @deque/cauldron-styles
+```
+
+#### yarn
+
+```js
+yarn add @deque/cauldron-react @deque/cauldron-styles
+```
+
+#### Peer Dependencies
+
+There are a few additional packages that you will need to install:
+
+- `react` at `16.x` or higher
+- `react-dom` at `16.x` or higher
+
+### Setup
+
+Once installed, you will need to add include cauldron's styles somewhere in the top level of your application. You will also need to include the required font family of Roboto with something like [google fonts](https://fonts.google.com/specimen/Roboto) or [fontsource](https://fontsource.org/fonts/roboto):
```

Comment:
could we show an example of how we generally do this? (aka the npm package and importing specific weight CSS)

Reply:
I didn't want to be _too_ descriptive since it's dependent on how your application is setup and may differ, but I updated this to at least include a baseline example of using fontsource.